app/src/conferences/models.py

```diff
@@ -235,18 +235,18 @@ class Meta:
     bus = models.ForeignKey(Bus, null=True, blank=True, on_delete=models.SET_NULL)
 
     # Day 1 (Coming)
-    accomodation_day_1 = models.BooleanField(default=False)
     dinner_1 = models.BooleanField(default=False)
```

Comment:
I think that we should change name of this field to `dinner_day_1` and the remaining analogously

Reply:
Or remove it from `accommodation_day_x`.